docs/blog/posts/bids-validator-2.0.md

```diff
@@ -0,0 +1,398 @@
+        deno
+    end
+
+    specification -. interpreted .-> regex
+    schema.json ---> deno
+```
+
+<caption>The BIDS schema is used as an authoritative,
+machine-readable source for the specification text and tools wishing
+to work with BIDS terms.
+Node.js, Python and Deno validators coexisted in a single repository.
+</caption>
+</figure>
+
+Recently, the [bids-standard/bids-specification][] repository has hosted the Markdown
+source code of the specification and the [bidsschematools][] package.
+This package compiles the schema from its YAML source into a single JSON blob,
+which is used for rendering the specification document and producing the
+canonical `schema.json` file that can be reused.
+
+In the [bids-standard/bids-validator][] repository, the Python and Javascript validators have
+continued to coexist, along with a new validator that is written to apply the
+schema, and targeting the [Deno] Javascript runtime.
+
+#### v1.14.7
+
+<figure>
+
+```mermaid
+graph LR;
+    subgraph bids-specification;
+        markdown ---> specification
+        schema --> bidsschematools --> specification & schema.json
+    end
+    subgraph "bids-validator @ v1.14.7";
+        regex["filename patterns"] --> Node.js
+        Node.js --> web
+        deno
+        python
+    end
+
+    specification -. interpreted .-> regex
+    schema.json ---> deno
+    bidsschematools ----> python
+```
+
+<caption>
+To prepare for separation, the Python validator dependency on the
+shared regular expressions was removed.
+</caption>
+</figure>
+
+In BIDS Validator v1.14.7, the Python validator was rewritten to depend
+on the [bidsschematools][] package, and no longer uses the hand-written
+regular expressions. At this point, the filename validation of the Python
+validator matches that of the Deno (schema) validator.
+
+#### v1.14.8
+
+<figure>
+
+```mermaid
+graph LR;
+    subgraph bids-specification;
+        markdown ---> specification
+        schema --> bidsschematools --> specification & schema.json
+    end
+    subgraph "bids-validator @ v1.14.8";
+        regex["filename patterns"] --> Node.js
+        Node.js --> web/legacy
+        deno --> web
+    end
+    subgraph "python-validator > v1.14.7";
+        python
+    end
+
+    specification -. interpreted .-> regex
+    schema.json ---> deno
+    bidsschematools ----> python
+```
+
+<caption>The Python validator was moved into its own repsitory.
+Its versioning and development are no longer tied to the Javascript
+validators.
+</caption>
+</figure>
+
+After the release of v1.14.7, the Python validator was
+removed from the [bids-standard/bids-validator] repository and
+relocated to the [bids-standard/python-validator] repository.
+Its development, if any takes place, will be independent of
+the Javascript validator(s).
+Its filename validation will be updated with new releases of [bidsschematools].
+
+The legacy validator was moved to <https://bids-standard.github.io/bids-validator/legacy/>.
```

Comment:
```suggestion
The legacy validator was moved to <https://bids-standard.github.io/legacy-validator/>.
```
caught by the dead link thingy

Reply:
Ah, I actually wanted to keep that, since it was where we moved things at that time. I think we should instead add a redirect: bids-standard/bids-validator#108

Reply:
Fixed.